apps/mark/frontend/src/setupTests.tsx

```diff
@@ -69,6 +69,9 @@ function mockSpeechSynthesis() {
 beforeEach(() => {
   mockSpeechSynthesis();
   fetchMock.mock();
+  // PointerEvent is unimplemented by js-dom
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (window as any).PointerEvent = Event;
```

Comment:
This is necessary because `contest_page` checks `instanceof PointerEvent` [here](https://github.com/votingworks/vxsuite/pull/4284/files#diff-f506ba67da9120a6f6d7f47a6988fec4355469efa7b5932ee71eb0aa3e4ad5b1R50). I was torn on this; feels like a trapdoor to require callers in other projects to add this to their test setup, and for some reason the resulting test failure from not having this polyfill results in an infinite loop and OOM errors in CI.

An alternative is to be less type safe:

```
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  return (nativeEvent as any).pointerType === '';
```

Also open to suggestions here.

Reply:
I think when I originally tried this approach I just checked for `'pointerType' in nativeEvent` which is both type-safe and doesn't require the `PointerEvent` class to be defined.

Reply:
I just did a little more research and experimentation and it looks like we can just use a combo of two top-level fields `event.detail` (which is '0' for keyboard or touch and '1' for mouse) and `event.touches` (which is an array for touch and not defined for mouse/keyboard). That way we don't need to access the `nativeEvent` at all